freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/model/common/Status.java

```diff
@@ -32,11 +37,11 @@
 
     public static final Collection<Status> AVAILABLE_STATUSES = List.of(AVAILABLE, MAINTENANCE_MODE_ENABLED);
 
-    public static final Collection<Status> REMOVABLE_STATUSES = List.of(AVAILABLE, UPDATE_FAILED, CREATE_FAILED, DELETE_FAILED,
+    public static final Collection<Status>  REMOVABLE_STATUSES = List.of(AVAILABLE, UPDATE_FAILED, CREATE_FAILED, DELETE_FAILED,
```

Comment:
still missing the failure states from here

Reply:
I added the newly added failed states (REPAIR_FAILED, UPSCALE_FAILED, DOWNSCALE_FAILED) to removable states now. Thanks!